qmail-newmrh.c

```diff
@@ -1,3 +1,4 @@
+#include <sys/types.h>
```

Comment:
Is there a pattern to when `<sys/types.h>` comes before `<sys/stat.h>` and when it comes after? I'd naively guess "always after", assuming the ordering is solely lexicographic.

Reply:
Should be that way, everything else is a mistake by me.